includes/discount-functions.php

```diff
@@ -929,7 +929,7 @@ function edd_get_discount_id_by_code( $code ) {
  * @return string $discounted_price Amount after discount
  */
 function edd_get_discounted_amount( $code, $base_price ) {
-	$amount      = 0;
+	$amount      = $base_price;
```

Comment:
For the notes (discussion happened on Slack), the function is returning the `$base_price` minus the `$rate` of the discount, so the returned value isn't the amount this item has been discounted, but the amount of the item AFTER discounts have been applied, so setting the amount of `0` initially means that if the `$code` passed in is not a valid discount, then the item price after discounts would be returned `0`, instead of the `$base_price`.